modules/ossm-automatic-gateway-injection.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-traffic-manage.adoc
+
+:_content-type: CONCEPT
+[id="ossm-automatic-gateway-injection_{context}"]
+= Enabling gateway injection
+
+Gateway configurations are applied to standalone Envoy proxies that are running at the edge of the mesh, rather than sidecar Envoy proxies running alongside your service workloads. Because gateways are Envoy proxies, you can configure {SMProductShortName} to automatically inject gateways similar to how you can automatically inject sidecars.
+
+Using automatic injection for gateways lets you deploy and manage gateways independent from the `ServiceMeshControlPlane` resource. This lets you manage your gateways with your user applications. Using auto-injection for gateway deployments gives developers full control over the gateway deployment, while also simplifying operations. When a new upgrade is available, or a configuration has changed, you restart the gateway pods to update them. This makes the experience of operating a gateway deployment the same as operating sidecars.
+
+[NOTE]
+====
+Injection is disabled by default for the `ServiceMeshControlPlane` namespace, for example `istio-system`. As a security best practice, deploy gateways in a different namespace from the control plane.
+====
+
+.Prerequisites
```

Comment:
Should this prerequisites section be part of a procedure module? Not sure why it's in a concept module.

Reply:
I broke this out into a procedure topic that uses callouts. Given how close with release deadline we are, I will create a new story to refactor the gateway content with the goal of creating procedural steps.